enterprise/models.py

```diff
@@ -723,20 +733,51 @@ def notify_enrolled_learners(self, catalog_api_user, course_id, users, admin_enr
                 )
             )
 
+        email_items = []
+        for user in users:
+            login_or_register = 'register' if isinstance(user, PendingEnterpriseCustomerUser) else 'login'
+            destination_url = destination_url.format(login_or_register=login_or_register)
+            email_items.append({
+                "user": utils.create_dict_from_user(user),
+                "enrolled_in": {
+                    'name': course_name,
+                    'url': destination_url,
+                    'type': 'course',
+                    'start': course_start,
+                },
+                "dashboard_url": dashboard_url,
+                "enterprise_customer_uuid": self.uuid,
+                "admin_enrollment": admin_enrollment,
+            })
+        return email_items
+
+    def notify_enrolled_learners(self, catalog_api_user, course_id, users, admin_enrollment=False):
```

Comment:
Don't name the notify_enrolled_learners task and this function the same thing, add _task if you are going to keep both items in 2 separate modules.